src/components/Board.vue

```diff
@@ -1,89 +1,129 @@
 <template>
-  <div id="game">
-    <div id="score">0</div>
+  <div id="game" ref="game">
+    <div id="score">
+      {{ this.$store.state.scoreStore.score }}
+    </div>
     <button class="button-start" @click="startGame">start</button>
-    <div id="touch-area">
-      <div id="good">
-        <div id="excellent">
-        </div>
+    <button class="button-start" style="margin-top: 80px" @click="stopGame">stop</button>
```

Comment:
Why do you use `style` if you have `class`. If you have bootstrap in you project, you should have classes like `mb-2`, `mb-3` etc.